integration-tests/src/main/java/io/quarkiverse/fury/it/FuryResources.java

```diff
@@ -25,6 +29,21 @@ public Boolean testSerializeFooRecord() {
         return foo1.equals(foo2);
     }
 
+    @GET
+    @Path("/bar")
+    public Boolean testSerializeBarRecord() {
+        Bar bar = new Bar(10, "abc");
+        Bar bar2 = (Bar) fury.deserialize(fury.serialize(bar));
+        Serializer serializer;
+        if (fury instanceof ThreadSafeFury) {
+            serializer = ((ThreadSafeFury) fury).execute(f -> f.getClassResolver().getSerializer(Bar.class));
+        } else {
+            serializer = ((Fury) fury).getClassResolver().getSerializer(Bar.class);
+        }
```

Comment:
@chaokunyang I just wonder why the `getSerializer` need to run in a thread?

So if we use `ThreadSafeFury`, what operations need to run in a thread or all the operations?

Reply:
It could be helpful to create a `FuryUtil` to help users to do this operation.

Reply:
All options needs to run in a thread. `ThreadSafeFury` is just a `Fury` pool, `Fury` share nothing between instances, we can't invoke `ThreadSafeFury#getClassResolver()` because it doesn't have a `ClassResolver`.

Reply:
> It could be helpful to create a `FuryUtil` to help users to do this operation.

Maybe in future, users don't needs to do such this operation